py/server/tests/test_udf_scalar_args.py

```diff
@@ -517,7 +517,7 @@ def f5(col1, col2: np.ndarray[np.int32]) -> bool:
             self.assertRegex(str(cm.exception), "f5: Expected .* got null")
 
         with self.subTest("f51"):
-            def f51(col1, col2: Optional[np.ndarray[np.int32]]) -> bool:
+            def f51(col1, col2: Optional[np.ndarray[np.int32]]) -> np.bool_:
```

Comment:
1. yes, because you need to create it, it is not the same as bool
2. an exception will be raised
3. added one specific test case in the UDF return test class.